Meta/lint-ports.py

```diff
@@ -374,10 +376,12 @@ def check_available_ports(from_table, ports):
         actual_ref = from_table[port]["dir_ref"]
         expected_ref = f"{port}/"
         if actual_ref != expected_ref:
-            print((
-                f'Directory link target in AvailablePorts.md for port {port} is '
-                f'incorrect, expected "{expected_ref}", found "{actual_ref}"'
-            ))
+            print(
+                (
+                    f'Directory link target in AvailablePorts.md for port {port} is '
+                    f'incorrect, expected "{expected_ref}", found "{actual_ref}"'
+                )
+            )
```

Comment:
This and a few others in this file look sus. why did we have `print (( f'blah' f'blah2' ))` ? Is that a normal pattern for multiline f-strings?

Reply:
😕 yeah this is strange, we can use triple quoted f-strings for multiple lines.